packages/lisk-core/src/helpers/logger.js

```diff
@@ -0,0 +1,6 @@
+const liskBunyan = require('@lisk/lisk-logger-bunyan');
+const config = require('../helpers/config');
+
+const logger = liskBunyan(config.components.logger);
+
+module.exports = logger;
```

Comment:
I don't like to have such helper files in the application. As explained in the issue already, I prefer to have factory pattern. You can get feedback from others on it.